demo/demo.html

```diff
@@ -7,7 +7,7 @@
 	<base href=".."></base>
 
 	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.8/angular.min.js"></script>
-	<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
+	<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>
```

Comment:
@pavelloz: How would that help?  That would look for the source file on the *local* server rather than the specified remote server.  You need a protocol.  If you wanted it to be protocol-agnostic, you'd need some code to pull in `location.protocol + "//code.query.com/…"` and that's both tedious and frivolous, especially given the HTTPS Google API loaded just before JQuery.